src/pages/StatisticsChart/index.tsx

```diff
@@ -2,6 +2,7 @@ import { ReactNode } from 'react'
 import 'default-passive-events'
 import { useTranslation } from 'react-i18next'
 import Content from '../../components/Content'
+import { isMainnet } from '../../utils/chain'
```

Comment:
`isMainnet` can be replaced by `IS_MAINNET` at https://github.com/nervosnetwork/ckb-explorer-frontend/blob/develop/src/constants/common.ts#L23 to avoid a function call